src/styles/mo-components/_fixed-width.scss

```diff
@@ -0,0 +1,9 @@
+.full-page-width { 
+    width: 100vw;
+    position: relative;
+    left: calc(-50vw + 50%);
```

Comment:
This is a really odd margin, how is it used?

Reply:
Good question! This is to center the inner div when the outer div is stretched the full width of the page. It moves the left edge of the inner div left half the width of the viewport (`-50vw`) and then moves it right half the width of the inner div (`+50%`). I recall trying other (seemingly simpler) approaches and this is the only one that worked, but I'm open to other suggestions!